tests/js/multiparser-css/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -285,13 +285,12 @@ export const Group = styled.div\`
   }
 \`;
 
-const yesFrame = (
-  ...args: Interpolation<ThemedStyledProps<{}, Theme>>[]
-) => css\`
-  \${ChatRoot}[data-frame="yes"] & {
-    \${css({}, ...args)}
-  }
-\`;
+const yesFrame = (...args: Interpolation<ThemedStyledProps<{}, Theme>>[]) =>
+  css\`
+    \${ChatRoot}[data-frame="yes"] & {
+      \${css({}, ...args)}
+    }
+  \`;
```

Comment:
Why it was changed?

Reply:
Before, if it's ``` css`` ``` and contains new lines, it print `css` on same line of arrow function. But problem is if ``` css`` ``` formatted as single line, the second format will change to print ``` css`` ``` on new line, so this became unstable.